gravitee-apim-console-webui/src/components/gio-notification-menu/gio-notification-menu.component.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject, timer } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+
+import { UserNotificationService } from '../../services-ngx/user-notification.service';
+import { UserNotification } from '../../entities/userNotification';
+
+@Component({
+  selector: 'gio-notification-menu',
+  template: require('./gio-notification-menu.component.html'),
+  styles: [require('./gio-notification-menu.component.scss')],
+})
+export class GioNotificationMenuComponent implements OnInit, OnDestroy {
+  public hasNotifications = false;
+  public userNotificationsCount = 0;
+  private unsubscribe$ = new Subject();
+  public userNotifications: UserNotification[];
+  isOpen: boolean;
+
+  constructor(public readonly userNotificationService: UserNotificationService) {}
+
+  public ngOnInit(): void {
+    timer(0, this.userNotificationService.getNotificationSchedulerInSeconds() * 1000)
```

Comment:
same maybe we can move this timer inside service and remove `getNotificationSchedulerInSeconds` 🤔